model_helen_comments.py

```diff
@@ -0,0 +1,127 @@
+"""Models and database functions for showme db."""
+
+from flask_sqlalchemy import SQLAlchemy
+
+db = SQLAlchemy()
+
+
+##############################################################################
+
+class Show(db.Model):
+    """Runway show details."""
+
+    __tablename__ = "shows"
+
+    show_id = db.Column(db.Integer,
+                        autoincrement=True,
+                        primary_key=True)
+    season = db.Column(db.String(6))
+    year = db.Column(db.Integer)
+    brand_id = db.Column(db.Integer,
+                         db.ForeignKey('brands.brand_id'))
+
```

Comment:
Even if you don't end up analyzing keywords, it may not hurt to create a review field or something like that with a column type of `Text`, because it can hold a large amount of text if you ever want to dump the review you scrape into here.